optimum/exporters/features.py

```diff
@@ -0,0 +1,744 @@
+        "resnet": supported_features_mapping(
+            "default",
+            "image-classification",
+            onnx="ResNetOnnxConfig",
+        ),
+        "roberta": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            "multiple-choice",
+            "token-classification",
+            "question-answering",
+            onnx="RobertaOnnxConfig",
+        ),
+        "roformer": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            "token-classification",
+            "multiple-choice",
+            "question-answering",
+            "token-classification",
+            onnx="RoFormerOnnxConfig",
+        ),
+        "segformer": supported_features_mapping(
+            "default",
+            "image-classification",
+            "semantic-segmentation",
+            onnx="SegformerOnnxConfig",
+        ),
+        "squeezebert": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "sequence-classification",
+            "multiple-choice",
+            "token-classification",
+            "question-answering",
+            onnx="SqueezeBertOnnxConfig",
+        ),
+        "t5": supported_features_mapping(
+            "default",
+            "default-with-past",
+            "seq2seq-lm",
+            "seq2seq-lm-with-past",
+            onnx="T5OnnxConfig",
+        ),
+        "vit": supported_features_mapping("default", "image-classification", "masked-im", onnx="ViTOnnxConfig"),
+        "xlm": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            "multiple-choice",
+            "token-classification",
+            "question-answering",
+            onnx="XLMOnnxConfig",
+        ),
+        "xlm-roberta": supported_features_mapping(
+            "default",
+            "masked-lm",
+            "causal-lm",
+            "sequence-classification",
+            "multiple-choice",
+            "token-classification",
+            "question-answering",
+            onnx="XLMRobertaOnnxConfig",
+        ),
+        "yolos": supported_features_mapping(
+            "default",
+            "object-detection",
+            onnx="YolosOnnxConfig",
+        ),
+    }
+
+    @staticmethod
+    def get_supported_features_for_model_type(
+        model_type: str, exporter: str, model_name: Optional[str] = None
+    ) -> FeatureNameToExportConfigDict:
+        """
+        Tries to retrieve the feature -> exporter backend config constructors map from the model type.
+
+        Args:
+            model_type (`str`):
+                The model type to retrieve the supported features for.
+            exporter (`str`):
+                The name of the exporter.
+            model_name (`str`, *optional*):
```

Comment:
```suggestion
            model_name (`str`, *optional*, defaults to `None`):
```

Reply:
Should not be specified when it is `None` I think.